manifests/install.pp

```diff
@@ -6,14 +6,30 @@
   ensure_packages($::oxidized::install_dependencies, { 'before' => 'Package[oxidized]' })
 
   if $facts.dig('os', 'family') == 'RedHat' {
-    $provider = 'scl_gem'
-    file { '/usr/local/bin/scl_gem':
-      ensure => 'file',
-      owner  => 'root',
-      group  => 'root',
-      mode   => '0755',
-      source => 'puppet:///modules/oxidized/scl_gem',
-      before => Package['oxidized'],
+    if versioncmp($::operatingsystemrelease, '8') >= 0 {
+      $provider = 'system_gem'
+      # libssh2 is not in repos for RHEL8
+      package{'libssh2':
```

Comment:
You can get libssh2 from EPEL: https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/l/

I would recommend adding `puppet/epel` as an optional dependency with default being `true` so a parameter like `manage_epel`. Once I upgrade to RHEL8 I will be using that at my site and we manage EPEL through Satellite so have to disable it from being included in modules to avoid conflicts.

Reply:
so it is my epel is being nasty ill swap this today if i look at the epel mirrors locally w/ yum i get libssh2 1.8 and libssh2-devel 1.9

Reply:
Just a thought how do we want to bring epel in? us Vox module or just push a yum repo

Reply:
Use the Vox module.